src/Entity/ApiDoc.php

```diff
@@ -34,6 +36,8 @@
  *   label_singular = @Translation("API Doc"),
  *   label_plural = @Translation("API Docs"),
  *   handlers = {
+ *
+ *     "storage" = "Drupal\Core\Entity\Sql\SqlContentEntityStorage",
```

Comment:
This should be the default. The only reason I can imagine it was added is because something was checking the definition and throwing an error because this value was missing.

Reply:
I removed it and tested it again and everything works as expected.